### ISSUE TITLE
[ISSUE] fmDNS warning creating and reloading zones: "dn_to_utf8(): INTL_IDNA_VARIANT_2003 is deprecated"

### ISSUE THREAD

**Issue by andrewhotlab** [WillyXJ/facileManager#481]
fM Version : **3.5.7**
fmdns Version : **3.4.1**

**In raising this issue, I confirm the following:**

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**Expected Behavior:**
Have the new zone dialog or the reload function displayed/act without errors.

**Actual Behavior:**
When creating a new zone, the dialog does not appear (thus it has become impossible to create new zones). See attached screenshot.

![Screenshot_2020-03-16_20-02-22](https://user-images.githubusercontent.com/26433166/76791553-735c4f80-67c1-11ea-9298-733b9b3b1c2f.png)

When reloading a zone, it displays the following error, but the process ends successfully.
```
Deprecated: idn_to_utf8(): INTL_IDNA_VARIANT_2003 is deprecated in [DocumentRoot]/fm-modules/fmDNS/functions.php on line 777
```
**Steps to reproduce:**
Click on `Zones > Forward > Add New`

_I was unable to reproduce the problem on a staging fm instance, thus I guess its someway related to PHP configuration of the production web server. Both of them runs Apache httpd 2.4.41 and PHP 7.2.26 (and PHP intl extension) on FreeBSD 11.3.
I'd be grateful if someone can suggest me what to check._

**Comment by WillyXJ** [comment]
The usage of idn_to_utf8() in fmDNS does not specify a variant so it's relying on the default or server's configuration.  I see INTL_IDNA_VARIANT_2003 was deprecated in PHP 7.2, but they did not change the default so you're getting those warnings (see https://bugs.php.net/bug.php?id=75609).

Until a fix is implemented in fmDNS to explicitly define the variant, you can disable the errors in the Settings which should allow you to manage your zones.

**Comment by andrewhotlab** [comment]
Thank you, but maybe I'm missing the right setting... I already have these lines in both servers' php.ini file:
```
display_errors = Off
display_startup_errors = Off
track_errors = Off
```

**Comment by WillyXJ** [comment]
This should be done in the facileManager settings as shown in this screenshot. This may also be the difference between your production and pre-production environments.

<img width="717" alt="Screen Shot 2020-03-16 at 3 21 12 PM" src="https://user-images.githubusercontent.com/1309591/76800723-dfc35880-6799-11ea-87c4-bcd2bfc87929.png">

**Comment by WillyXJ** [comment]
This is now fixed in [fmDNS 4.0-beta1](http://www.facilemanager.com/modules/fmdns/?t=3) and later. Final release will come out later.

**Comment by WillyXJ** [comment]
[fmDNS 4.0.0](http://www.facilemanager.com/modules/fmdns/) has been released.
